### PR TITLE
Makes the blacklist more clear

### PR DESCRIPTION
**Pull request by Dimiikou**
Appends modifiers to the blacklist instead of prepend

### CODE REVIEW

File: src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java [hunk 1]
```diff
@@ -79,6 +80,12 @@ public static void onBlacklistInfo(ClientChatReceivedEvent e) {
         if (!name.equals(target)) return;
 
         if (type == ModifyBlacklistType.OUTLAW) {
+            if (reason.contains("[Vogelfrei]")) {
+                TextUtils.error("Der Spieler ist bereits Vogelfrei.");
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java:84]: "vogelfrei" should be written in lower case

File: src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java [hunk 2]
```diff
@@ -97,6 +110,15 @@ public static void onBlacklistInfo(ClientChatReceivedEvent e) {
         p.sendChatMessage("/bl set " + target + " " + kills + " " + price + " " + reason);
     }
 
+    //Removes all known Modifiers
+    private static String removeModifiers(String reason) {
+        reason = reason.replace(" [Yobannoe dno]", "") // Removes Yobbanoe dno modifier
```
Comment (paulzhng) [inline src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java:115]: can be replaced with a Regex removing everything in brackets, e.g. `(:?\[|\().+(:?\]|\))`

**Review by paulzhng — CHANGES_REQUESTED** [review]

File: src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java (hunk 1 above)
Comment (Dimiikou) [reply src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java:84]: contains is case sensitive, so why should we wake it lower case, if we append the reason with an uppercase letter

Comment (paulzhng) [reply src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java:84]: I'm talking about the message `TextUtils.error(#)`

File: src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java (hunk 2 above)
Comment (Dimiikou) [reply src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java]: Done

File: src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java (hunk 1 above)
Comment (Dimiikou) [reply src/main/java/de/fuzzlemann/ucutils/commands/faction/badfaction/blacklist/ModifyBlacklistCommand.java]: Also done

**Review by paulzhng — APPROVED** [review]

**Pull request merged**